vscode/package.json

```
@@ -574,6 +574,13 @@
         "icon": "$(copy)",
         "title": "Copy Cody Extension Version"
       },
+      {
+        "command": "cody.command.logCharacterCounters",
```

```suggestion
        "command": "cody.test.logCharacterCounters",
```

Nit: matches the internal command below